src/palace.pyx

```diff
@@ -2232,90 +2275,117 @@ cdef class ReverbEffect(BaseEffect):
         self.slot.apply_effect(self.impl)
 
     @property
     def modulation_depth(self) -> float:
+        """Modulation depth, from 0.0 to 1.0."""
         return self.properties.modulation_depth
 
     @modulation_depth.setter
     def modulation_depth(self, value: float) -> None:
+        if value < 0.0 or value > 1.0:
+            raise ValueError(f'invalid modulation depth: {value}')
         self.properties.modulation_depth = value
         self.impl.set_reverb_properties(self.properties)
         self.slot.apply_effect(self.impl)
 
     @property
     def air_absorption_gain_hf(self) -> float:
+        """Air absorption gain, from 0.892 to 1.0."""
         return self.properties.air_absorption_gain_hf
 
     @air_absorption_gain_hf.setter
     def air_absorption_gain_hf(self, value: float) -> None:
+        if value < 0.892 or value > 1.0:
+            raise ValueError(f'invalid air absorption gain hf: {value}')
         self.properties.air_absorption_gain_hf = value
         self.impl.set_reverb_properties(self.properties)
         self.slot.apply_effect(self.impl)
 
     @property
     def hf_reference(self) -> float:
+        """HF reference, from 1000.0 to 20000.0."""
         return self.properties.hf_reference
 
     @hf_reference.setter
     def hf_reference(self, value: float) -> None:
+        if value < 1000.0 or value > 20000.0:
+            raise ValueError(f'invalid hf reference: {value}')
         self.properties.hf_reference = value
         self.impl.set_reverb_properties(self.properties)
         self.slot.apply_effect(self.impl)
 
     @property
     def lf_reference(self) -> float:
+        """LF reference, from 20.0 to 1000.0."""
         return self.properties.lf_reference
 
     @lf_reference.setter
     def lf_reference(self, value: float) -> None:
+        if value < 20.0 or value > 1000.0:
+            raise ValueError(f'invalid lf reference: {value}')
         self.properties.lf_reference = value
         self.impl.set_reverb_properties(self.properties)
         self.slot.apply_effect(self.impl)
 
     @property
     def room_rolloff_factor(self) -> float:
+        """Room rolloff factor, from 0.0 to 10.0."""
         return self.properties.room_rolloff_factor
 
     @room_rolloff_factor.setter
     def room_rolloff_factor(self, value: float) -> None:
+        if value < 0.0 or value > 10.0:
+            raise ValueError(f'invalid room rolloff factor: {value}')
         self.properties.room_rolloff_factor = value
         self.impl.set_reverb_properties(self.properties)
         self.slot.apply_effect(self.impl)
 
     @property
     def decay_hf_limit(self) -> bool:
+        """Decay HF limit, either TRUE or FALSE."""
         return self.properties.decay_hf_limit
 
     @decay_hf_limit.setter
     def decay_hf_limit(self, value: bool) -> None:
+        if value != True or value != False:
+            raise ValueError(f'invalid decay hf limit: {value}')
         self.properties.decay_hf_limit = value
```

Comment:
I'm not sure about this, maybe just `self.properties.decay_hf_limit = bool(value)` is enough.